mailer.js

```diff
@@ -321,7 +334,8 @@ module.exports = function (log) {
         link: link,
         oneClickLink: oneClickLink,
         supportUrl: this.supportUrl,
-        supportLinkAttributes: this._supportLinkAttributes()
+        supportLinkAttributes: this._supportLinkAttributes(),
+        passwordChangeUrl: this._initiatePasswordChange()
```

Comment:
opened https://github.com/mozilla/fxa-auth-mailer/issues/94 so we can merge this and cut strings